### PR TITLE
Bam2fastq fixes and refactorings

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

File: RoddyCore/src/de/dkfz/roddy/config/Configuration.java [hunk 1]
```diff
@@ -362,6 +362,7 @@ public File getSourceToolPath(String tool) throws ConfigurationError {
         return toolPath;
     }
 
+    /** The the actual path to the copy of the tool on the execution host (which can be local or remote). */
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/config/Configuration.java]: The the

File: RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy [hunk 2]
```diff
@@ -246,44 +246,48 @@ abstract class ExecutionService implements BEExecutionService {
         }
     }
 
+    /** Create somehow valid submission IDs.
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy]: Create low-level issue, this should somehow be in JobManager / BatchEuphoria

File: dist/bin/develop/helperScripts/findPluginLibraries.groovy [hunk 3]
```diff
@@ -67,23 +78,28 @@ List<File> collectAllJars(List<String> pluginSearchDirectories, String pluginDir
 }
 
 
-// This script is used to extract plugin jar files from the various plugin directories.
-// The plugin directories come from the roddy configuration
-// The dependencies are stored within the buildinfo.txt files for the specified plugin
+// This script is used to the JARs for plugins on which a query plugin depends. Multiple plugin directories, configured in, e.g., the
```
Comment (dankwart-de) [inline dist/bin/develop/helperScripts/findPluginLibraries.groovy]: to find the JARs?

Comment (dankwart-de) [reply dist/bin/develop/helperScripts/findPluginLibraries.groovy]: query plugin? is that right?

File: RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy (hunk 2 above)
Comment (vinjana) [reply RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy]: See https://github.com/TheRoddyWMS/BatchEuphoria/issues/118

File: dist/bin/develop/helperScripts/findPluginLibraries.groovy (hunk 3 above)
Comment (vinjana) [reply dist/bin/develop/helperScripts/findPluginLibraries.groovy]: Yes. Updated the text.

**Pull request merged**